src/pydap/client.py

```diff
@@ -55,9 +55,11 @@
 import pydap.parsers.das
 import pydap.parsers.dds
 import pydap.parsers.dmr
-from pydap.handlers.dap import UNPACKDAP4DATA
+from pydap.handlers.dap import UNPACKDAP4DATA, DAPHandler
 from pydap.lib import DEFAULT_TIMEOUT as DEFAULT_TIMEOUT
 
+from .net import create_session
```

Comment:
In this _the_ .net?

Reply:
this is `pydap.net`. In this scenario, it refers to the module net with `pydap` project. It is equivalent to

```python
from pydap.net import create_session
```

I am partial to `.net`, although pydap for the most part, uses the other alternative approach (you can see there on line 59 `from pydap.lib import ....` instead of `from .lib import ...`

Reply:
OK, I get it.